corehq/apps/sms/views.py

```diff
@@ -1039,11 +1050,15 @@ def backend_class_name(self):
     def ignored_fields(self):
         return [
             'give_other_domains_access',
+            'phone_numbers',
         ]
 
     @property
     @memoized
     def backend_class(self):
+        # For now, only allow superusers to create/edit non-Telerivet backends
+        if not self.is_superuser and self.backend_class_name != "TelerivetBackend":
```

Comment:
Both are correct, no? Basically all we're saying is that if your not a superuser, then you can't edit anything except a Telerivet backend (which is what the code says).  And the comment is saying the same thing - only superusers can edit non-Telerivet backends.